assignments/prototype-refactor.js

```diff
@@ -6,4 +6,215 @@ Prototype Refactor
 
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
-*/
+*/ 
+//Create GameObject Constructor
+    function GameObject(GOattributes){
+        this.createdAt = GOattributes.createdAt;
+        this.name = GOattributes.name;
+        this.dimensions = GOattributes.dimensions;
```

Comment:
What exactly is the `dimensions` attribute, and how would you get it's attributes individually?